tedana/decomposition/pca.py

```diff
@@ -204,12 +205,11 @@ def tedpca(
         f"Computing PCA of optimally combined multi-echo data with selection criteria: {algorithm}"
     )
     data = data_oc[mask, :]
-
-    data_z = ((data.T - data.T.mean(axis=0)) / data.T.std(axis=0)).T  # var normalize ts
-    data_z = (data_z - data_z.mean()) / data_z.std()  # var normalize everything
+    data_z = standardize_signal(data.T, detrend=True, standardize="zscore_sample").T
+    # data_z = (data_z - data_z.mean()) / data_z.std()  # var normalize everything
```

Comment:
I'm just not sure what the point of z-scoring based on the overall mean and SD does.